### ISSUE TITLE
Using Deck.gl with normalized mercator projection and 256 tiles (HERE maps)

### ISSUE THREAD

**Issue by madrussa** [visgl/deck.gl#4147]
Hi, I'm trying to use Deck.gl with [Here Maps](https://developer.here.com/documentation/maps/dev_guide/index.html) which uses normalized mercator projection and 256 tiles. 

This projection and tile difference to Deck.gl is why I think my overlay is offset on the y axis when zoomed out. The further I zoom out the further north my layer travels.
![image](https://user-images.githubusercontent.com/1470257/72360419-d6f5ce00-36e7-11ea-8c60-3ea101582ed3.png)
 Whilst zoomed all the way in, it's pretty accurate:
![image](https://user-images.githubusercontent.com/1470257/72360476-ea089e00-36e7-11ea-8329-a6d9fa4081c1.png)

Code used to create HERE maps overlay which is appended to the map:
```javascript
const overlay = new H.map.layer.CanvasLayer((gl, params) => {
  const { zoom: zm, bounds } = params;
  const centerPoint = bounds.getCenter();
  const zoom = zm - 1; // Better when zoomed in with minus one.

  if (!this.deck) {
    this.deck = new Deck({
      gl,
      initialViewState: {
        longitude: centerPoint.lng,
        latitude: centerPoint.lat,
        zoom,
        pitch: 0,
        bearing: 0
      },
      controller: false,
      layers: this.layers
    });
  }

  this.deck.setProps({
    viewState: {
      latitude: centerPoint.lat,
      longitude: centerPoint.lng,
      zoom,
      pitch: 0,
      bearing: 0
    },
  });
  this.deck.redraw();
  return H.map.render.RenderState.DONE;
}, {
  contextType: 'webgl'
});
```

Is there any setting or conversion I can perform to help align the layer at all zoom levels?

**Comment by Pessimistress** [comment]
It might worth checking if the base map is using the same fov as deck.gl. Deck.gl's fov is `Math.atan(1 / 3) * 2`

**Comment by madrussa** [comment]
I've tried adjusting the field of view for HERE maps, unfortunately it's hard to tell if it's making a difference. There's an undocumented CameraData object I've manipulated to set the FOV to 64 (originally 70 so assuming its using integers), but it didn't have any effect. 

The layer only moves when zooming, so if I zoom out to say 10 and move the map around the layer is fixed to the terrain but is offset on the Y axis further north to where it should be. 

Thank you for the suggestion!
